cmd/crates/soroban-spec-typescript/ts-tests/package.json

```diff
@@ -2,12 +2,21 @@
 	"private": true,
 	"type": "module",
 	"scripts": {
+		"postinstall": "curl -X POST \"http://localhost:8000/soroban/rpc\" && npm run fund  && npm run deploy && npm run bindings || { echo \"Make sure you're running standalone RPC network on localhost:8000\" && exit 1; }",
+		"fund": "./soroban config identity fund",
+		"bindings:custom-types": "./soroban contract bindings typescript --contract-id $(cat contract-id-custom-types.txt) --output-dir ./node_modules/test-custom-types --overwrite",
+		"bindings:hello-world": "./soroban contract bindings typescript --contract-id $(cat contract-id-hello-world.txt) --output-dir ./node_modules/test-hello-world --overwrite",
+		"bindings": "npm run bindings:custom-types && npm run bindings:hello-world",
+		"deploy:custom-types": "(./soroban contract deploy --wasm ../../../../target/wasm32-unknown-unknown/test-wasms/test_custom_types.wasm) > contract-id-custom-types.txt",
+		"deploy:hello-world": "(./soroban contract deploy --wasm ../../../../target/wasm32-unknown-unknown/test-wasms/test_hello_world.wasm) > contract-id-hello-world.txt",
+		"deploy": "npm run deploy:custom-types && npm run deploy:hello-world",
```

Comment:
Notice that this now adds CI tests for the networked versions of CLI commands `contract bindings` and `contract deploy` directly to the `soroban-tools` repo, rather than needing `system-test` for these tests.

If we also add a simple `invoke` test, then we'll have duplicated system-test with a more-reliable, more-maintainable, three-minute CI workflow.